### ISSUE TITLE
Create a Footer for the Signup/Signin Page

### ISSUE THREAD

**Issue by ezioda004** [UdacityFrontEndScholarship/quiz-app#26]
Create a Footer as shown in [this picture](https://raw.githubusercontent.com/parulbindal/quiz-app/master/src/assets/images/Sign%20In%20Page%20Detailed.jpg)

How to get started? Read our [project setup instructions](https://github.com/UdacityFrontEndScholarship/quiz-app/blob/master/CONTRIBUTING.md#project-setup-instructions).
Fork the project, navigate to directory quiz-app => src => Component => Footer.js

All you need to do is write HTML inside the `<footer> </footer>` element.

Note: 
1. If you use any self closing tags such as `<img>`, `<input>`, `<br>`
Make sure you always close them like this `<img />`, `<input />`, `<br />`

2. If you add a `class` to an element example 
```
<div  class = "nav navbar"> 
    <!-- rest of your code -->
</div>
```
Make sure you use **className** instead of **class**. Example

```
<div  className = "nav navbar"> 
    <!-- rest of your code -->
</div>
```

**You only need to create the elements, please don't style them yet.** I'll create different issue for that after this one is resolved.

Don't worry about links, just add `href = "#"` for now. We'll do the rest!

This is fairly easy and straightforward to do, please comment on this issue and I'll assign it to you.

**Comment by bhshkh** [comment]
Please assign this issue to me

**Comment by ezioda004** [comment]
@bhshkh You can work on this, make sure you complete by tomorrow positively or we may have to assign it somebody else. Any queries regarding, contact any of the co-mods or me.

**Comment by lalitmee** [comment]
Assigned to @bhshkh. :+1: 
But follow the guides in the issue as mentioned. Okk. :innocent:

**Comment by Amir22010** [comment]
i want to contribute to this section

**Comment by kadumuri1994** [comment]
@Amir22010 , thanks for your interest. But I think this issue has been assigned to @bhshkh and I guess he is already working on it. We will be uploading the issues every now and then. Wait for it :)

**Comment by ezioda004** [comment]
@bhshkh Any updates? Please update me by 8 o'clock tonight on slack. If you don't I'll have to reassign this to someone else.

**Comment by kadumuri1994** [comment]
@SukhjinderArora yes, sure. But let's see if @bhshkh  can provide any update by 8 o' clock as @ezioda004  mentioned.

**Comment by ezioda004** [comment]
@SukhjinderArora Assigning it to you now, please do it by tomorrow. If you have any questions, feel free to ask.

**Comment by ezioda004** [comment]
Resolved #39 
Closing this issue.
